lab1/relazione/src/sections/acquisizione.tex

```diff
@@ -0,0 +1,121 @@
+%! Author = alida
+%! Date = 22/11/2024
+
+% Preamble
+\documentclass[@SRC@/main]{subfiles}
+\graphicspath{@IMAGES@}
+
+% tutti i pacchetti usati vanno nel main
+
+% Document
+\begin{document}
+
+\section{Apparato sperimentale e svolgimento} \label{sec:acquisizione}
+  Il circuito in \textit{Fig.\!\!\! 1} è stato realizzato su una millefori, ed è stato alimentato mediante un $alimentatore\, di\, bassa\, tensione$, erogante una differenza di potenziale di +5V. 
```

Comment:
Hai usato la math mode per fare il testo corsivo?
Per quello hanno fatto \textit{}

Edit: sono tornato qui dopo aver visto che sotto l'hai usato, ti pesava cambiarlo?? (a questo punto inizio a prendere in considerazione l'ipotesi che ci sia un buon motivo sotto)